# Type: docs
# Subtype: D2
# English Summary: Add comments for super method
# Chinese Summary: 为super方法添加注释
# Entry Number: 4

# === Git Diff Content ===

diff --git a/java/idea-ui/src/com/intellij/openapi/roots/ui/configuration/projectRoot/BaseStructureConfigurable.java b/java/idea-ui/src/com/intellij/openapi/roots/ui/configuration/projectRoot/BaseStructureConfigurable.java
index 09492ff..6d37bd6 100644
--- a/java/idea-ui/src/com/intellij/openapi/roots/ui/configuration/projectRoot/BaseStructureConfigurable.java
+++ b/java/idea-ui/src/com/intellij/openapi/roots/ui/configuration/projectRoot/BaseStructureConfigurable.java
@@ -339,6 +339,7 @@ public abstract class BaseStructureConfigurable extends MasterDetailsComponent i
 
   protected class MyRemoveAction extends MyDeleteAction {
     public MyRemoveAction() {
+      //noinspection Convert2Lambda
       super(new Condition<Object[]>() {
         @Override
         public boolean value(final Object[] objects) {
